Bottom: 5a0317e1a673b028fd81e6dd6ddb9ac21bc2730d
Top:    2ccbc6390623a665bcccefdd44a62d335a9996e8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:11:44 -0500

Refresh of mariadb-fix-bind-address

---
diff --git a/roles/mariadb/tasks/main.yml b/roles/mariadb/tasks/main.yml
index 966c9a93..4436e593 100644
--- a/roles/mariadb/tasks/main.yml
+++ b/roles/mariadb/tasks/main.yml
@@ -10,6 +10,14 @@
 - name: Config MariaDB server
   template: src=ansible.cnf.j2 dest={{ mariadb_server_config_dir }}/ansible.cnf
             owner=root group=root mode=0644 backup=yes
+- name: Set host mysqld bind-address
+  template:
+    src: mariadb-bind-address.j2
+    dest: '{{ mariadb_server_config_dir }}/mariadb-bind-address.cnf'
+    owner: root 
+    group: root 
+    mode: 0644 
+    backup: yes
 - name: Enable and start MariaDB service
   service: name=mariadb enabled=yes state=started
 - name: Check previous installation
@@ -20,14 +28,3 @@
 - name: Open the firewall
   firewalld: service=mysql permanent=true state=enabled
   when: mariadb_open_firewall
-- name: Set host mysqld bind-address
-  template:
-    src: mariadb-bind-address.j2
-    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
-  register: bind_cnf
-- name: Restart MariaDB service
-  service:
-    name: mariadb
-    enabled: yes
-    state: restarted
-  when: bind_cnf.changed
